src/BullOak.Repositories.EventStore.Test.Integration/Specification/SaveEventStream.feature

```diff
@@ -66,3 +66,13 @@ Scenario: Write after a soft deleted stream should succeed
 	And I load my entity
 	Then the load process should succeed
 	And HighOrder property should be 9
+
+Scenario: Write after a soft delete by event for a stream should succeed
```

Comment:
The scenario implementation does not test what the title says, as I can see.
What it tests is that after soft-deleted event stream is considered empty.

This is a valid test, but need another scenario that would test write after a soft delete event, i.e. what happens when we try to re-use that stream.
Need